components/engine_tirocks/src/properties/range.rs

```diff
@@ -182,7 +182,7 @@ impl RangeProperties {
 impl From<SizeProperties> for RangeProperties {
     fn from(p: SizeProperties) -> RangeProperties {
         let mut res = RangeProperties::default();
-        for (key, size_handle) in p.index_handles.into_map() {
+        for (key, size_handle) in p.prop_indexes.into_map() {
             let range = RangeOffsets {
                 size: size_handle.offset,
```

Comment:
Why the offset is size?

Reply:
For `SizeProperties`, the `offset` is accumulation of the `size`. For `RangeOffsets`, `size` is the total bytes of raw KVs from the file start.